modules/display.js

```diff
@@ -0,0 +1,18 @@
+
+import { booksList } from '/index.js'; // Module is evaluated!
+
+const booksSection = document.getElementById('books-list');
+export default function displayBooks() {
```

Comment:
- [ ] Great function implementation, but as the project required, kindly change that to an arrow function so it can follow the ES6 best practices.

Reply:
This change has been done well